GliaWidgets/Sources/ViewModel/Chat/ChatViewModel+ChoiceCards.swift

```diff
@@ -0,0 +1,63 @@
+import Foundation
+import SalemoveSDK
+
+extension ChatViewModel {
+    func sendChoiceCardResponse(_ option: ChatChoiceCardOption, to messageId: String) {
```

Comment:
nit, since the param name is not a part of the signature, I think it should not be omitted.